src/components/MapView.jsx

```diff
@@ -1,37 +1,37 @@
 import 'leaflet/dist/leaflet.css';
-import React, {useContext, useEffect, useState} from 'react';
-import {MapContainer, TileLayer} from 'react-leaflet';
-import {useDispatch, useSelector} from 'react-redux';
-import {useTranslation} from 'react-i18next';
+import React, { useContext, useEffect, useState } from 'react';
+import { MapContainer, TileLayer } from 'react-leaflet';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
 
 import api from '../api/apiInstance';
-import {basicHeaders} from '../config/apiConfig';
-import {NotificationActions} from '../redux/actions/notificationActions';
-import {unsafeApiError} from '../utilities/utilities';
-import {MapContext} from './contexts/MapContextProvider';
+import { basicHeaders } from '../config/apiConfig';
+import { NotificationActions } from '../redux/actions/notificationActions';
+import { unsafeApiError } from '../utilities/utilities';
+import { MapContext } from './contexts/MapContextProvider';
```

Comment:
In our `.prettierrc` file there is property `"bracketSpacing": false` - please check if you use proper formatter in your VSCode -> Ctrl+Shift+P -> `Format Document with...`, use prettier or run command in terminal `npm run prettier:components`

Reply:
Ok, I'll check that next time.